### PR TITLE
Update tfm for Rails 5.2 and Ruby 2.5

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
[test rpm]

File: packages/foreman/tfm/tfm.spec [hunk 1]
```diff
@@ -79,11 +79,11 @@ Obsoletes: %{scl_prefix}rubygem-foreman_param_lookup < 0.3.0-3
 Obsoletes: %{scl_prefix}rubygem-foreman_reserve < 0.1.8.3-6
 Obsoletes: %{scl_prefix}rubygem-foreman_snapshot < 0.1.0-5
 Obsoletes: %{scl_prefix}rubygem-foremancli < 1.0-10
+Obsoletes: %{scl_prefix}rubygem-jquery_pwstrength_bootstrap_4 < 1.2.2-5
```
Comment (ekohl) [inline packages/foreman/tfm/tfm.spec]: What's the reason to move this around?

**Comment by ehelms** [comment]
[test rpm]

File: packages/foreman/tfm/tfm.spec (hunk 1 above)
Comment (ehelms) [reply packages/foreman/tfm/tfm.spec]: None -- it was not present when I did this branch originally and the rebase moved it. I'll put it back

**Pull request merged**